internal/msg/errormsg.go

```diff
@@ -221,3 +221,11 @@ const (
 	// InsightsReportError indicates failure push to insights.
 	InsightsReportError = "unable to report result to insights"
 )
+
+// npm related messages
+const (
+	// NpmEmptyURLError indicates a missing URL value
+	NpmEmptyURLError = "url of scoped registry at index %d is empty"
+	// NpmTooManyDefaultRegistry indicates that there is multiple registry without scope
+	NpmTooManyDefaultRegistry = "too many registries (%d) are without scope"
```

Comment:
The lingo suggests that there exists a number of scopeless registries that is OK to have. What's that?
The variable name is actually on point.

```suggestion
	NpmTooManyDefaultRegistry = "multiple default (scopeless) registries defined"
```

or

```suggestion
	NpmTooManyDefaultRegistry = "only one default (scopeless) registry is allowed"
```

or something else along those lines 😄